### ISSUE TITLE
building 3.8.1 offline fails on Ruxit

### ISSUE THREAD

**Issue by avasseur-pivotal** [cloudfoundry/java-buildpack#331]
(I added some more log to display the configuration uri)

https://download.ruxit.com/agent/paas/cloudfoundry/java/index.yml
rake aborted!

Invalid version '': missing component
/Users/avasseur/s2/tofino/jbp/java-buildpack/lib/java_buildpack/util/tokenized_version.rb:151:in `validate'
/Users/avasseur/s2/tofino/jbp/java-buildpack/lib/java_buildpack/util/tokenized_version.rb:42:in `initialize'

The index.yml on ruxit.com has some spaces that looks suspect here

```
 curl -X GET https://download.ruxit.com/agent/paas/cloudfoundry/java/index.yml
---
  1.101.170: https://download.ruxit.com/agent/paas/Dynatrace-OneAgent-linux-java-1.101.170.20160830-115303.zip
  1.99.199: https://download.ruxit.com/agent/paas/ruxit-agent-linux-java-1.99.199.20160817-132932.zip
  1.97.216: https://download.ruxit.com/agent/paas/ruxit-agent-linux-java-1.97.216.20160704-154931.zip
```

**Comment by nebhale** [comment]
The offline buildpack cannot make an external connection to the Ruxit repository.  It is fixed to only using [the `1.95.149` version](https://github.com/cloudfoundry/java-buildpack/blob/v3.8.1/config/ruxit_agent.yml#L18) that was packaged inside when it was created.  Looking at the binary downloaded from GitHub, the following files are contained within it:

```
resources/cache/https%3A%2F%2Fdownload.ruxit.com%2Fagent%2Fpaas%2Fcloudfoundry%2Fjava%2Findex.yml.cached
resources/cache/https%3A%2F%2Fdownload.ruxit.com%2Fagent%2Fpaas%2Fcloudfoundry%2Fjava%2Findex.yml.etag
resources/cache/https%3A%2F%2Fdownload.ruxit.com%2Fagent%2Fpaas%2Fcloudfoundry%2Fjava%2Findex.yml.last_modified
resources/cache/https%3A%2F%2Fdownload.ruxit.com%2Fagent%2Fpaas%2Fruxit-agent-linux-java-1.95.149.20160607-131029.zip.cached
resources/cache/https%3A%2F%2Fdownload.ruxit.com%2Fagent%2Fpaas%2Fruxit-agent-linux-java-1.95.149.20160607-131029.zip.etag
resources/cache/https%3A%2F%2Fdownload.ruxit.com%2Fagent%2Fpaas%2Fruxit-agent-linux-java-1.95.149.20160607-131029.zip.last_modified
```

**Comment by nebhale** [comment]
Fixed [by the Ruxit team](https://cloudfoundry.slack.com/archives/java-buildpack/p1473780722000049).
